### DISCUSSION TITLE
"rust-lang/book" 레포지토리와의 동기화 방법

### DISCUSSION THREAD

**Discussion by jeongchaeuk** [rust-kr/doc.rust-kr.org#17] [Q&A] [answered]
내용이나 예제 소스가 "rust-lang/book"(영문 원본)과 다른 부분이 있던데
"rust-lang/book" 레포지토리의 최신 내용들은 어떻게 여기 러스트 한글 레포지토리에 반영되는건가요?

**Comment by rinthel** [comment, marked as answer]
안녕하세요, 늦게 답변 드립니다.
영문 md를 지워가면서 한글로 번역하는 식이다 보니, merge 등등의 자동적인 방법은 사용하지 않았고,
원본 repo의 기준점 commit을 잡은 다음 번역 안된 md는 덮어쓰기를, 번역된 md에는 이전 기준점 commit과 최신 commit간의 diff를 보고 변경점을 수동 반영하는 식으로 진행했습니다.

**Reply by rinthel** [reply]
음 지금 제가 잠시 확인해본 바로는, 가장 마지막에 업데이트된 기준점이 https://github.com/rust-lang/book/commit/77092be 이 커밋인거 같은데요, 벌써 2년 전이군요 😞 시간될 때 최신 버전 반영을 해두겠습니다-
